.eslintignore

```diff
@@ -0,0 +1,6 @@
+# don't ever lint node_modules
+node_modules
+# don't lint build output (make sure it's set to your correct build folder name)
+dist
+# don't lint nyc coverage output
+coverage
```

Comment:
We don't have either `coverage` or `dist` folder - build is created in `build` directory, so we can change `dist` to `build`.

Reply:
Ok, done.